kubernetes/kustomize/config/backend/nginx.conf

```diff
@@ -14,4 +14,12 @@ server {
 	    port_in_redirect off;
 		autoindex on;
 	}
+
```

Comment:
If this is a common issue for backend nginx configuration, you should seperate this change and change the nginx configuration located in the docker folder as well.

Reply:
I will try it with docker and if it works, let the backend-httpd pod use the file from container looks reasonable.